### DISCUSSION TITLE
Portainer help

### DISCUSSION THREAD

**Discussion by Chrissica06** [codeproject/CodeProject.AI-Server#164] [General]
I loaded CPAI via portianer.  I only need OCR to load.  but everytime a reboot the original setup loads.  How do I fix this?

**Comment by MikeLud** [comment]
What version are you running, if I remember correctly V2.6.5 had issues with saving your config.

**Reply by Chrissica06** [reply]
That appears to be the issue.   Pull kept pulling 2.6.5 not 2.8.   Got the newest pull and I can restart all day long with no issues

**Comment by ChrisMaunder** [comment]
We can't provide support for containerisation technologies outside of Docker. There are far too many for us to keep track of and we need to focus our resources on what we know and can control. My only suggestion is to run CodeProject.AI outside of a container, ensure it all works as expected, and try and work backwards to see what the difference is.

**Reply by Chrissica06** [reply]
Thank you.  I tried loading docker desktop for hours and couldn't get it working.   I've been running blueiris and cpai on a windows VM for years till my lack of Linux knowledge killed my proxmox server.  Starting over trying to run it on something a little lighter than windows.  I guess it was more of a community question. Didnt expect a developer reply.  Great product.  Love it.

**Reply by ChrisMaunder** [reply]
What OS are you using? Docker on Ubuntu/Debian is pretty straight forward (https://docs.docker.com/engine/install/ubuntu/) and our docs (https://codeproject.github.io/codeproject.ai/install/running_in_docker.html) provide the command lines to create / run an image.

Happy to provide whatever help I can on that end.
